ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist

```diff
@@ -13,15 +13,15 @@
 	<key>JavaScriptInjectionDelay</key>
 	<integer>100</integer>
 	<key>KillSwitchEnabled</key>
-	<false/>
+	<true/>
```

Comment:
was this intentional?

Reply:
Yes the TLFResources.bundle was included in the release zip I got from the TeaLeaf repo, no mods from me just plopped it in

Reply:
Currently killswitch is turned off by default.